server/webserver_test.go

```diff
@@ -0,0 +1,31 @@
+package server
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/livepeer/go-livepeer/core"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSetOrchestratorPriceInfo(t *testing.T) {
```

Comment:
Can we add tests for when `pricePerUnitStr` and/or `pixelsPerUnitStr` are not integers i.e. if `strconv.ParseInt()` returns an error?

Reply:
fixed in 894375e